ostd/src/sync/rcu/mod.rs

```diff
@@ -422,6 +433,47 @@ unsafe fn delay_drop<P: NonNullPtr + Send>(pointer: NonNull<<P as NonNullPtr>::T
     });
 }
 
+/// A wrapper to delay calling destructor of `T` after the RCU grace period.
+///
+/// Upon dropping this structure, a callback will be registered to the global
+/// RCU monitor and the destructor of `T` is delayed until the callback.  
```

Comment:
Sorry I failed to catch the tense here... "will be registered" is used before, so: "is delayed" -> "will be delayed"